examples/00-fluent/frozen_rotor_workflow.py

```diff
@@ -0,0 +1,424 @@
+)
+
+volute_mesh = examples.download_file(
+    "volute.msh.h5",
+    "pyfluent/examples/pump-volute",
+)
+
+################################################################################################################
+# Define Constants
+# ==============================================================================================================
+
+rho_water = 998.2  # kg/m^3
+mu_water = 0.001002  # kg/(m.s)
+g = 9.81  # m/s^2
+# Impeller speed
+impeller_speed = 1450  # rpm
+# Convert to rad/s
+impeller_speed_rad = impeller_speed * 2 * math.pi / 60  # rad/s
+
+################################################################################################################
+# Launch Fluent with solver mode and print Fluent version
+# ==============================================================================================================
+
+solver = pyfluent.launch_fluent(
+    mode="solver",
+    ui_mode="gui",
+    processor_count=4,
+    cleanup_on_exit=True,
+)
+print(solver.get_fluent_version())
+
+
+################################################################################################################
+# Read, append the mesh files
+# ==============================================================================================================
+
+# Read Impeller mesh file
+solver.settings.file.read_mesh(file_name=impeller_mesh)
+# Append Volute mesh file
+solver.settings.mesh.modify_zones.append_mesh(file_name=volute_mesh)
+
+################################################################################################################
+# Display the mesh
+# ==============================================================================================================
+
+# Access the graphics object
+graphics = solver.settings.results.graphics
+
+# Create a mesh object and configure its settings
+mesh_object = solver.settings.results.graphics.mesh.create(name="mesh-1")
+mesh_object.surfaces_list = [
+    "inlet",
+    "mass-flow-inlet-11",
+    "interface-impeller-outlet",
+    "interface-volute-inlet",
+    "blade",
+    "impeller-hub",
+    "impeller-shroud",
+    "inblock-hub",
+    "inblock-shroud",
+    "volute-inlet-wall",
+    "volute-wall",
+]
+mesh_object.options.edges = True
+
+
+# Set the hardcopy format for saving the image
+graphics.picture.driver_options.hardcopy_format = "png"
+
+graphics.views.restore_view(view_name="front")
+mesh_object.display()
+graphics.views.auto_scale()
+graphics.picture.save_picture(
+    file_name="pump-volute-mesh.png",
+)
+
+
+# %%
+# .. image:: ../../_static/pump-volute-mesh.png
+#    :align: center
+#    :width: 650px
+#    :height: 400px
+#    :alt: Pump Volute Mesh
+
+################################################################################################################
+# Set Unit for Angular Velocity
+# ==============================================================================================================
+
+# Setting unit for angular velocity to rev/min
```

Comment:
@abhishekchitwar I believe that these two comments should be more precise in terms of the user's rationale and the outcome of calling `set_units()` rather than the currently vanilla comment. The call sets the display units in Fluent but has no impact on the solver nor on PyFluent.